### DISCUSSION TITLE
BLAS=0

### DISCUSSION THREAD

**Discussion by ricardoborges** [abetlen/llama-cpp-python#1053] [Q&A]
Hey guys, what i'm doing wrong here... it's a windows 11 machine, rtx 3050. I tried a lot of things to install llama-cpp-python for GPU as written in readme, but when I execute the code it's always print "... BLAS = 0 | ..."

am i missing something?

**Comment by rrrusst** [comment]
For the command
`set CMAKE_ARGS="-DLLAMA_CUBLAS=on"`

On Windows command prompt, you need to quote the whole line
`"set CMAKE_ARGS=-DLLAMA_CUBLAS=on"`

I had the same issues as you. Followed the steps to the letter but always had BLAS = 0. Turns out you need to quote the whole line for the set CMAKE_ARGS instead of just quoting the args (thanks to [this post](https://github.com/abetlen/llama-cpp-python/discussions/871) )

**Reply by ricardoborges** [reply]
Thank you bro!
